apps/web/src/components/DropdownItem/index.tsx

```diff
@@ -29,6 +29,11 @@ export default function DropdownItem({ children, copyValue, onClick }: DropdownI
     setLastCopiedId(id);
   }, [id, setLastCopiedId]);
 
+  const onClickHandler = useCallback(() => {
+    setOpen(false);
+    if (onClick) onClick();
```

Comment:
```suggestion
    onClick?();
```

p sure you can do this instead